examples/kubernetes/deepspeech/Dockerfile

```diff
@@ -0,0 +1,97 @@
+FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+RUN apt-get update
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        wget \
+        git \
+        python \
+        python-dev \
+        python-pip \
+        python-wheel \
+        python-numpy \
+        libcurl3-dev  \
+        ca-certificates \
+        gcc \
+        sox \
+        libsox-fmt-mp3 \
+        htop \
+        nano \
+        swig \
+        cmake \
+        libboost-all-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        liblzma-dev \
+        locales \
+        pkg-config \
+        libsox-dev
+
+
+RUN apt-get install -y python3 python3-pip
+RUN pip3 install 'tensorflow-gpu==1.6.0' pandas python_speech_features pyxdg progressbar2 scipy
+# >> START Configure Tensorflow Build
+RUN apt-get install -y software-properties-common
+RUN add-apt-repository ppa:git-core/ppa
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
+RUN apt-get install -y git-lfs
+RUN git lfs install
+
+RUN git clone https://github.com/mozilla/DeepSpeech.git
+WORKDIR /DeepSpeech/
+RUN git reset --hard e00bfd0f413912855eb2312bc1efe3bd2b023b25
+# >> START Configure Tensorflow Build
+RUN apt-get install -y software-properties-common
+RUN add-apt-repository ppa:git-core/ppa
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
+RUN apt-get install -y git-lfs
+RUN git lfs install
```

Comment:
Installing git-lfs results in downloading the language model binary (1.8 GB), which is not needed for the benchmarks, and takes a lot of time and space. I suggest to make sure if simply not installing git-lfs will do, or if setting GIT_LFS_SKIP_SMUDGE=1 is required.